gci/client.py

```diff
@@ -49,21 +47,13 @@ class GCIAPIClient(object):
   """
 
   def __init__(self, auth_token=None,
-               url_prefix='https://codein.withgoogle.com/',
-               debug=False):
+      url_prefix='https://codein.withgoogle.com/'):
     self.url_prefix = urlparse.urljoin(url_prefix, 'api/program/current/')
     self.headers = {
         'Authorization': 'Bearer %s' % auth_token,
         'Content-Type': 'application/json',
     }
 
-    if debug:
```

Comment:
please revert this.  debug is not enabled.  And is not enabled at https://github.com/coala/community/blob/master/gci/students.py#L30